assets/src/components/routeLadder.tsx

```diff
@@ -177,6 +177,27 @@ export const groupByPosition = (
   }
 }
 
+const runNotSharedByAnotherVehicle = (vehiclesAndGhosts: VehicleOrGhost[]) => (
+  vehicle: Vehicle
+): boolean => {
+  if (vehicle.runId === null) {
```

Comment:
When do we ever have a null run id?

Reply:
I don't have a great sense, but it seems like they come in occasionally, but this is also just about dealing with the type definition.